servicemesh-install/topics/install.adoc

```diff
@@ -27,301 +71,568 @@ The 3scale Istio Adapter is deployed and configured in the custom resource file.
+        traceSampling: 100.0
+
+      kiali:
+       dashboard:
+          user: admin
+          passphrase: admin
+      tracing:
+        enabled: true
 ----
 
-The following example illustrates the minimum required to install the control plane.  This minimal example custom resource deploys the CentOS-based community Istio images.
+[[custom-resource-parameters]]
+== Custom resource parameters
 
-.Minimum example istio-installation.yaml
+The following examples illustrate use of the supported custom resource parameters for {ProductName} and the tables provide additional information about supported parameters.
+
+[IMPORTANT]
+====
+The resources you configure for {ProductName} with these custom resource parameters, including CPUs, memory, and the number of pods, are based on the configuration of your OpenShift cluster. Configure these parameters based on the available resources in your current cluster configuration.
+====
+
+=== Istio global example
+
+[NOTE]
+====
+In order for the 3scale Istio Adapter to work, `disablePolicyChecks` must be `false`.
+====
 
 [source,yaml]
 ----
-apiVersion: "istio.openshift.com/v1alpha1"
-kind: "Installation"
-metadata:
-  name: "istio-installation"
+  istio:
+    global:
+      hub: `maistra/` if `deployment_type=origin` and `openshift-istio-tech-preview/` if `deployment_type=openshift`
+      tag: 0.10.0
+      proxy:
+        resources:
+          requests:
+            cpu: 100m
+            memory: 128Mi
+          limits:
+            cpu: 500m
+            memory: 128Mi
+      mtls: false
+      disablePolicyChecks: true
+      policyCheckFailOpen: false
+      imagePullSecrets:
+        - MyPullSecret
 ----
 
-[[custom-resource-parameters]]
-== Custom resource parameters
-
-The following tables list the supported custom resource parameters for {ProductName}.
 
 .General parameters
 |===
-|Parameter |Values |Description | Default
+|Parameter |Description |Values |Default
 
-|`deployment_type`
-|`origin`, `openshift`
-|Specifies whether to use Origin (community) or {product-title} (product) default values for undefined parameter values.
-|`origin`
-|===
-
-.Istio parameters
-|===
-|Parameter |Values |Description | Default
+|`disablePolicyChecks`
+|This boolean indicates whether to enable policy checks
+|`true`/`false`
+|`true`
 
-|`authentication`
+|`policyCheckFailOpen`
+|This boolean indicates whether traffic is allowed to pass through to the Envoy sidecar when the Mixer policy service cannot be reached
 |`true`/`false`
-|Whether to enable mutual authentication.
 |`false`
 
-|`community`
+|`tag`
+|The tag that the operator uses to pull the Istio images
+|A valid container image tag
+|`0.10.0`
+
+|`hub`
+|The hub that the operator uses to pull Istio images
+|A valid image repo
+|`maistra/` if `deployment_type=origin` and `openshift-istio-tech-preview/` if `deployment_type=openshift`
+
+|`mTLS`
+|This controls whether to enable Mutual Transport Layer Security (mTLS) between services by default
```

Comment:
I think mutual TLS is better than Mutual Transport Layer Security

Reply:
The problem with acronyms is that there are usually multiple possible definitions.  Acronym Finder has seven for MTLS.  Spelling it out helps anyone who doesn't know what the acronym stands for (especially folks who have English as a second language.)